### PR TITLE
chore: add typings

### PR DESCRIPTION
**Pull request by Shigma**
**Summary**

Declaration files for `vuepress`.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe: _typings_

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

N/A

If adding a **new feature**, the PR's description includes:

N/A

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by f3ltron** [comment]
I know it's been a while but it look's really great infos. I will make a review to be sure it is up to date with the master version

File: yarn.lock [hunk 1]
```diff
@@ -1556,6 +1707,7 @@
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/@types/hash-sum/-/hash-sum-1.0.0.tgz#838f4e8627887d42b162d05f3d96ca636c2bc504"
 
+<<<<<<< HEAD
```
Comment (kefranabg) [inline yarn.lock]: @f3ltron I think you've committed some conflicts in here 😛

Comment (f3ltron) [reply yarn.lock]: ups i will correct it

**Comment by meteorlxy** [comment]
Currently we have an unofficial types definition package: [vuepress-types](https://www.npmjs.com/package/vuepress-types)

**Pull request closed without merging**